controllers/roleController.js

```diff
@@ -0,0 +1,148 @@
+import models from "../models";
+import constants from "../helpers/constants";
+import { isString, isEmpty } from "../helpers/funcValidators";
+
+const {
+  OK,
+  INTERNAL_SERVER_ERROR,
+  CONFLICT,
+  NOT_FOUND,
+  BAD_REQUEST,
+  CREATED
+} = constants.statusCode;
+const { Role, Permission, User } = models;
+class RoleController {
+  static async create(req, res) {
+    try {
+      const { name, description } = req.body;
+      if (isEmpty(name)) {
+        return res.status(BAD_REQUEST).json({
+          message: `Name is required`
+        });
+      }
+      if (!isString(name) || !isString(description)) {
+        return res.status(BAD_REQUEST).json({
+          message: `{${name}} and {${description}} can only be alphabetic characters`
+        });
+      }
+      const role = await Role.create({
+        name: name.toUpperCase(),
```

Comment:
@niomwungeri-fabrice Can we have duplicate roles with the same names?

Reply:
Hi @oesukam, no we can't we have constraints in the database that prevent it.